docs/NFF-0.1-spec.norg

```diff
@@ -0,0 +1,875 @@
+    - A delimiting modifier can _only_ occur at the beginning of the line (arbitrary {# whitespace}
+      (but nothing else!) may precede it)
+    - A delimiting modifier must consist of three or more consecutive modifiers of the same type
+    - It must be followed by an immediate {# line ending} (without any extra {# whitespace})
+
+**** Weak Delimiting Modifier
+     This modifier uses the `-` character and immediately closes the current heading level.
+     @code norg
+     * Heading level 1
+       Text under first level heading.
+
+     ** Heading level 2
+        Text under second level heading.
+        ---
+
+       Text under first level heading again.
+     @end
+
+**** Strong Delimiting Modifier
+     This modifier uses the `=` character and immediately closes all heading levels.
+     @code norg
+     * Heading level 1
+       Text under first level heading.
+
+     ** Heading level 2
+        Text under second level heading.
+        ===
+
+     Text belonging to no heading level (i.e. to the document's root).
+     @end
+
+**** Horizontal Rule
+     This modifier uses the `_` character and simply renders a horizontal line. It does _NOT_
+     affect the heading level but immediately terminates a {# paragraph}.
+     @code norg
+     * Heading level 1
+       Text under first level heading.
+       ___
+       This is a new paragraph separated from the previous one by a horizontal line.
+       This text still belongs to the first level heading.
+     @end
+
+** Range-able Detached Modifiers
+   Range-able detached modifiers can occur in two forms:
+   - As a single detached modifier they affect:
+   -- The next `paragraph_segment` which becomes the /title/
+   -- The `paragraph` following that which becomes the /content/
+   - As a pair of two detached modifiers in which case:
+   -- The next `paragraph_segment` still becomes the /title/
+   -- But the content continues until the "closing" detached modifier is found
+
+   The closing modifier has the same rules as an opening detached modifier except it /must/ be
+   directly succeeded by a {# line ending}.
+
+*** Definitions
+    @code norg
+    $ Single Definition
+    Optional definition content.
+    @end
+
+    @code norg
+    $$ Multiple Definition
+    Content of the definition.
+
+    Which scans up to the closing modifier.
+    $$
+    @end
+
+*** Footnotes
+    @code norg
+    ^ Single Footnote
+    Optional footnote content.
+    @end
+
+    @code norg
+    ^^ Multiple Footnote
+    Content of the footnote.
+
+    Which scans up to the closing modifier.
+    ^^
+    @end
+
+*** Insertions
+    @code norg
+    = Insertion arg1 arg2 ...
+    @end
+    Depending on the kind of insertion, a different number of arguments can be set/required. These
+    arguments follow the same rules as those of {# Tags}.
+    +TODO: not sure I really like this. Tags are separate from detached modifiers, we shouldn't
+    draw parallels here+
```

Comment:
I disagree. Simply because having different kinds of rule sets simple to avoid this parallel would be confusing.